### PR TITLE
Cover Scrapy 2.1 in the release notes

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: docs/news.rst [hunk 1]
```diff
@@ -3,6 +3,135 @@
...
+
+*   The ``MultiValueDict``, ``MultiValueDictKeyError``, and ``SiteNode``
+    classes have been removed from :mod:`scrapy.utils.datatypes`
+    (:issue:`4400`)
+
+
+Deprecations
+~~~~~~~~~~~~
+
+*   :meth:`Spider.make_requests_from_url
```
Comment (kmike) [inline docs/news.rst:49]: It is deprecated a long time ago; I wonder if we can use a different wording.

Comment (Gallaecio) [reply docs/news.rst]: I’ve moved it to the bug fixes section, which I guess it is technically what we did, fix the deprecation reporting.

File: docs/news.rst [hunk 2]
```diff
@@ -3,6 +3,145 @@
...
+
+*   The ``REDIRECT_MAX_METAREFRESH_DELAY`` setting is no longer supported, use
+    :setting:`METAREFRESH_MAXDELAY` instead (:issue:`4385`)
+
+*   The :class:`~scrapy.downloadermiddlewares.chunked.ChunkedTransferMiddleware`
+    middleware has been removed, including the entire
+    :class:`scrapy.downloadermiddlewares.chunked` module; chunked transfers
+    work out of the box (:issue:`4431`)
+
+*   The ``spiders`` property has ben removed from
```
Comment (kmike) [inline docs/news.rst:36]:
```suggestion
*   The ``spiders`` property has been removed from
```

**Comment by Gallaecio** [comment]
I though about listing the exceptions and the circumstances under which they arise, but since none of them were documented before (and I think some of them cannot even be caught), I think these 2 paragraphs may be enough.

File: docs/news.rst [hunk 3]
```diff
@@ -3,6 +3,153 @@
...
+Backward-incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*   :exc:`AssertionError` exceptions triggered by :ref:`assert <assert>`
+    statements have been replaced by new exception types, to support running
+    Python in optimized mode (see :option:`-O`) without changing Scrapy’s
+    behavior in any unexpected ways.
+
+    If you catch an :exc:`AssertionError` exception from Scrapy, update your
+    code to catch the corresponding new exception.
```
Comment (kmike) [inline docs/news.rst:25]: I think there is one exception from this - scrapy contracts still raise an AssertionError subclass. So technically, someone could be catching AssertionError from scrapy contracts. I'm not sure when it is the case, so maybe the current wording is fine.

**Pull request merged**